spec/diagrams/115_AnalysingAggregation.plantuml

```diff
@@ -8,7 +8,7 @@ end title
 participant "LinkVis" as LinkVis
 participant "ALAP://v1/provide-acceptance-data-of-link-endpoint" as ALAP
 participant "<i>RequestForProvidingAcceptanceDataCausesReadingLtpStructure</i>" as LTPs
-participant "MWDI://core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/ltp-augment-1-0:ltp-augment-pac?fields=original-ltp-name" as WireInterfaceName
+participant "MWDI://core-model-1-4:network-control-domain=cache/control-construct={mount-name}/logical-termination-point={uuid}/ltp-augment-1-0:ltp-augment-pac?fields=(original-ltp-name;external-label)" as WireInterfaceName
```

Comment:
Can we update alias: wireInterfaceName in this line, but now Aggregation is meant to retrieve data for both Air and Wire Interfaces.

Reply:
Participant alias changed from wireInterfaceName to LtpDesignation